tests/nvme/064

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2024 Hannes Reinecke (SUSE) <hare@suse.de>
+#
+# Test nvme fabrics controller ANA failover during I/O
+
+. tests/nvme/rc
+
+DESCRIPTION="test admin commands on disabled paths"
+
+requires() {
+	_nvme_requires
+	_have_loop
+	_have_fio
+	_require_nvme_trtype_is_fabrics
+}
+
+set_conditions() {
+	_set_nvme_trtype "$@"
+}
+
+test() {
+	local -a ports
+	local port
+	local ns
+
+	echo "Running ${TEST_NAME}"
+
+	_setup_nvmet
+
+	_nvmet_target_setup
+
+	_nvme_connect_subsys
+
+	ns=$(_find_nvme_ns "${def_subsys_uuid}")
+	[ -z "${ns}" ] && return 1
+	nvme nvm-id-ctrl "/dev/${ns}"
+	nvme nvm-id-ns "/dev/${ns}"
```

Comment:
Everytime `nvm-id-ctrl` or `nvm-id-ns` is touched this test will break. I think it would be better to test for expected values.

Reply:
To make this test a bit future proof, it really should check for specific fields, as we do in nvme/059:

```shell
 nvme_awupf=$(nvme id-ctrl /dev/"${ctrl_dev}" | grep awupf | awk '{ print $3}')
```

BTW, I still think we should start using the json output for nvme-cli together with `jq`, e.g:

```shell
nvme id-ctrl /dev/nvme0 --output json | jq .awupf
```